docs/get-started/air-gap.md

```diff
@@ -0,0 +1,369 @@
+    
+*   If there's no BIOS setting, WiFi can usually be disabled almost as easily on laptops by opening them up to remove, or disconnect the leads to, the WiFi card.
+
+#### Put Ubuntu in [Airplane mode](https://help.ubuntu.com/stable/ubuntu-help/net-wireless-airplane.html)
+
+This will disable any Bluetooth services as well as WiFi, and shows as an Airplane on Ubuntu & other GNOME desktops as an airplane icon in the upper right corner of the screen.
+
+With Airplane Mode always engaged, you would need the obvious Internet cable plugged in to have any network access (unlike WiFi which can often be connected by accident).
+
+#### Add your computer's WiFi MAC address to the blacklist on your Internet router
+
+Some routers maintain a list of MAC addresses which will not be given an IP address by DHCP, which isolates them from the Internet unless that network interface is configured manually.
+
+Therefore, you can [find your WiFi MAC
+address](https://help.ubuntu.com/stable/ubuntu-help/net-macaddress.html.en)
+and add it to your
+router's blacklist: usually in its DNS, DHCP, or LAN settings.
+
+### Congratulations, your Air Gap Environment is complete!
+
+You now have a safe place you can use for your [Secure Transaction Workflow](./secure-workflow).
+
+## Option 2: Install your Air Gap environment on a persistent USB drive
+
+:::caution
+
+Linux veterans only! (otherwise please [follow option 1](#option-1-install-your-air-gap-environment-on-a-standalone-computer))
+
+:::
+
+This option may suit more demanding users, especially those:
+
+* who travel a lot and need to maintain their Cardano operations "on the road";
+* who need the convenience of booting in a Air Gap environment which has direct access to all their files on the host computer (as you would when booting off from an installer USB drive);
+* who, instead of using a USB drive to transfer unencrypted files in & out of the Air Gap, would rather use that same USB drive to store them with encryption while providing the Cardano CLI for use on any machine supporting the same boot method;
+* who want to make backups or their keys, passwords and other records from their Air Gap environment directly to the host computer.
+
+If this appeals to you, and you don't mind following a more complilcated
+and error-prone installation procedure, you might want to install the
+Air Gap environment on a bootable USB drive instead.  You can then boot a
+computer from this drive to have access to your secure resources and `cardano-cli`
+while isolating that computer from the Internet as well as any malicious software that might be installed on that computer.
+
+This loosely documented configuration has been called the **Frankenwallet**, with
+separate instructions at this link which mostly follow the procedure above and include
+semantics for using your bootable USB environment in secure & blockchain
+workflow:
+
+* **[The Frankenwallet](https://cosd.com/frankenwallet)**
```

Comment:
Might be better to add a page explaining different address types within dev portal instead

Reply:
I don't understand how the suggestion relates to this part of the content.  That [Option 2](https://github.com/rphair/developer-portal/blob/air-gap%2Bsecure-workflow/docs/get-started/air-gap.md#option-2-install-your-air-gap-environment-on-a-persistent-usb-drive) section is there so readers know they can use Cardano securely without a second machine, i.e. that it's at least possible (even if they don't choose this method) to build a secure transaction hosting environment on a USB device instead.  What does that have to do with "different address types"?

Reply:
My bad - I was a bit hasty with this comment (as community came up with the term `franken addresses` recently, which is unrelated to what you mention)